### PR TITLE
[IMP] beesdoo_product: make suggested price rounding configurable

### PR DESCRIPTION
**Pull request by tfrancoi**
Allow to configure the rounding of suggested price
choose the method and the precision of the rounding on the category

### CODE REVIEW

File: beesdoo_product/models/beesdoo_product.py [hunk 1]
```diff
@@ -451,6 +457,17 @@ def _check_margin(self):
             if product.profit_margin < 0.0:
                 raise UserError(_("Percentages for Profit Margin must >= 0."))
 
+    def _round(self, price):
+        self.ensure_one()
+        # Use default value 0.05 and "HALF_UP"
+        # in case someone erase the value on the category
+        # Keep default value on the field to make it explicit to the end user
```
Comment (robinkeunen) [inline beesdoo_product/models/beesdoo_product.py:468]: Can't we rather set the rounding method and precision to required ?

**Review by robinkeunen — APPROVED** [review]
Small comments. Tests would be nice, we'll do it when splitting the module and move it to the oca.

File: beesdoo_product/models/beesdoo_product.py (hunk 1 above)
Comment (robinkeunen) [inline beesdoo_product/models/beesdoo_product.py:468]: Can't we rather set the rounding method and precision to required ?

**Review by robinkeunen — CHANGES_REQUESTED** [review]

**Comment by polchampion** [comment]
@robinkeunen Testé sur oufticoop-test et OK

**Review by robinkeunen — APPROVED** [review]

**Comment by vdewulf** [comment]
Pol has documented this new feature here:
https://doc.it4socialeconomy.org/books/configurations/page/ajouter-sur-la-fiche-article-les-labels-prix-suggere-et-dangerosite

**Pull request merged**